src/components/SearchInput.tsx

```diff
@@ -1,63 +1,48 @@
-import { useEffect, useRef, useState } from 'react';
+import { useRef, useState } from 'react';
 
-import styled, { type AnyStyledComponent, css } from 'styled-components';
+import styled, { type AnyStyledComponent } from 'styled-components';
 
 import { layoutMixins } from '@/styles/layoutMixins';
 
-import { IconName } from '@/components/Icon';
-import { IconButton } from '@/components/IconButton';
-import { Input, type InputProps } from '@/components/Input';
+import { Icon, IconName } from '@/components/Icon';
+import { Input, type InputProps, InputType } from '@/components/Input';
+import { IconButton } from './IconButton';
```

Comment:
nit: for consistency, feels like we should keep the abs path import `@/components/IconButton`